### PR TITLE
Website/ More article sorting

### PR DESCRIPTION
**Pull request by DavidBachmann**
Workaround for **custom** article and subgroup sorting by using an `importance` integer.

Sort subgroups by `importance`:
image

No subgroups mean it's sorted alphabetically.
image

### CODE REVIEW

**Comment by stefanhar** [comment]
We could use aggregation and sorting in elasticsearch to do this, could make the frontend more snappy.
This lays the ground work for that if we want to go that route

File: libs/api/domains/cms/src/lib/services.ts [hunk 1]
```diff
@@ -0,0 +1,437 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
+import { getLocalizedEntries } from './contentful'
+import { logger } from '@island.is/logging'
+import { ApolloError } from 'apollo-server-express'
+import * as types from './generated/contentfulTypes'
+import { Article, mapArticle } from './models/article.model'
```
Comment (stefanhar) [inline libs/api/domains/cms/src/lib/services.ts:6]: Could this file be here by a misstake?

**Review by stefanhar — CHANGES_REQUESTED** [review]

File: libs/api/domains/cms/src/lib/services.ts (hunk 1 above)
Comment (DavidBachmann) [reply libs/api/domains/cms/src/lib/services.ts]: Definitely is

**Review by stefanhar — APPROVED** [review]
LGTM

**Review by haukurh — APPROVED** [review]

**Review by arnifreyrs — APPROVED** [review]
LGTM

**Review by darrikonn — APPROVED** [review]

**Pull request merged**